docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md

```diff
@@ -145,20 +152,21 @@ The current iteration of the Cluster API Provider AWS relies on credentials
 being present in your environment. These then get written into the cluster
 manifests for use by the controllers.
 
-If you used `clusterawsadm` to set up IAM resources for you then you can run
-these commands to prepare your environment.
-
-Your `AWS_REGION` must already be set.
+E.g.
 
 ```bash
-export AWS_CREDENTIALS=$(aws iam create-access-key \
-  --user-name bootstrapper.cluster-api-provider-aws.sigs.k8s.io)
-export AWS_ACCESS_KEY_ID=$(echo $AWS_CREDENTIALS | jq .AccessKey.AccessKeyId -r)
-export AWS_SECRET_ACCESS_KEY=$(echo $AWS_CREDENTIALS | jq .AccessKey.SecretAccessKey -r)
+export AWS_REGION=us-east-1 # This is used to help encode your environment variables
+export AWS_ACCESS_KEY_ID=<your-access-key>
+export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
+export AWS_SESSION_TOKEN=<session-token> # If you are using Multi-Factor Auth.
 ```
 
-If you did not use `clusterawsadm` to provision your user, you will need to set
-these environment variables in your own way.
+**Note**: The credentials used must have the appropriate permissions for use by the controllers.
```

Comment:
So that account also need to have permissions for control plane policy?

Reply:
The permissions are needed for `capa-controller-manager` (and `capa-eks-control-plane-controller-manager` if EKS enabled). 
The credentials are used to populate these two secrets:
* `capa-eks-control-plane-controlplane-manager-bootstrap-credentials`
* `capa-manager-bootstrap-credentials`